jetty-core/jetty-session/src/main/java/org/eclipse/jetty/session/UnreadableSessionDataException.java

```diff
@@ -13,14 +13,14 @@
 
 package org.eclipse.jetty.session;
 
+import java.io.Serial;
```

Comment:
According to the javadoc for `Serial` this annotation is also supposed to be applied to the `readObject` and `writeObject` methods.